### PR TITLE
1381 check user registered

### PR DESCRIPTION
**Pull request by nasief**

### CODE REVIEW

**Comment by nasief** [comment]
[(1) django-users-plus - modify logic to check if the email entered exists before reseting the record and show an error message if it does not exist [Week43]](https://trello.com/c/cQEThfSc/1381-1-django-users-plus-modify-logic-to-check-if-the-email-entered-exists-before-reseting-the-record-and-show-an-error-message-if-it)

**Comment by nasief** [comment]
[(1) 53TWO - Fix change_password and reset_password pages UI, and make sure they are functioning as expected [Week43]](https://trello.com/c/HYefTK5H/1370-1-53two-fix-change-password-and-reset-password-pages-ui-and-make-sure-they-are-functioning-as-expected-week43)

File: accountsplus/forms.py [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
 
 
 class CaptchaForm(django.forms.Form):
     captcha = ReCaptchaField()
     username = django.forms.CharField()
+
+    def clean_username(self):
+        username = self.cleaned_data.get('username')
+        User = apps.get_model(getattr(settings, 'AUTH_USER_MODEL'))
+        exists = User.objects.filter(email=username).exists()
```
Comment (danaspiegel) [inline accountsplus/forms.py]: you don't need to store exists in a local variable. you can just use it in the if test below.

Comment (nasief) [reply accountsplus/forms.py]: Makes sense :D

**Pull request merged**